package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "breakjs": "^1.0.0",
     "classnames": "^2.2.5",
+    "lodash": "^4.17.5",
```

Comment:
i don't see lodash referenced any place? did you mean to remove it?

Reply:
would be great to stay lodash free in the distributed code if possible... we have been using it as `optionalDependency` for some of the Storybook story examples only so far...but trying to to stay es6 ;)
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore